src/Components/MovieTile/MovieTile.jsx

```diff
@@ -0,0 +1,18 @@
+import styles from './MovieTile.module.css';
+
+export const MovieTile = ({ imageUrl, movieName, releaseYear, relevantGenres, onClick }) => {
+  const handleClick = () => {
+    onClick();
+  };
+
+  return (
+    <div className={styles.container} onClick={handleClick}>
```

Comment:
A separate handler is usually defined only if there is some additional logic inside it.
Here you can pass `onClick` as is to the container:

```suggestion
  return (
    <div className={styles.container} onClick={onClick}>
```